### PR TITLE
[TIC-474] Add get/set functions for Active Org

### PR DESCRIPTION
**Pull request by itailevi98**
https://propelauth.atlassian.net/browse/TIC-474

### CODE REVIEW

File: src/org.ts [hunk 1]
```diff
@@ -9,3 +12,11 @@ export type OrgMemberInfo = {
 export type OrgIdToOrgMemberInfo = {
     [orgId: string]: OrgMemberInfo
 }
+
+export const setActiveOrgId = (orgId: string) => {
+    Cookies.set(ACTIVE_ORG_ID_COOKIE_NAME, orgId)
```
Comment (andrew-propelauth) [inline src/org.ts]: Could you also add secure and same-site: lax?

**Review by andrew-propelauth — APPROVED** [review]
one small change then lgtm!

File: src/org.ts (hunk 1 above)
Comment (itailevi98) [reply src/org.ts]: Added

**Review by andrew-propelauth — APPROVED** [review]
lgtm!

**Pull request merged**